crates/sp-domains/src/fraud_proof.rs

```diff
@@ -208,13 +208,13 @@ pub enum ProofDataPerExpectedInvalidBundle {
 
 #[derive(Debug, Decode, Encode, TypeInfo, PartialEq, Eq, Clone)]
 pub struct TrueInvalidBundleEntryFraudProof {
-    bad_receipt_hash: ReceiptHash,
-    domain_id: DomainId,
-    bundle_index: u32,
-    mismatched_extrinsic: OpaqueExtrinsic,
-    mismatched_extrinsic_index: u32,
-    extrinsic_inclusion_proof: Vec<Vec<u8>>,
-    proof_data: ProofDataPerExpectedInvalidBundle,
+    pub bad_receipt_hash: ReceiptHash,
```

Comment:
this could be part of the previous commit since it does not make any new changes